I really don't know what to put here

JokeOfAllTrades opened the pull request:
Hopefully you won't yell at me too long for not familiarizing myself with all this before I put my thumb in the pie.

JokeOfAllTrades committed:
A Test File is Created

During the initialization of the test class, a test file is created to run tests against.  It is deleted when the tests finish.

JokeOfAllTrades committed:
New test

Added Add Unit By Path tests.  Started on Scan Default Directories test

hexafluoride commented on SharpinitUnitTest/UnitRegistryTests.cs:
```diff
@@ -4,20 +4,33 @@
+            {
+                File.Delete(path);
+            }
+            FileStream unitFileStream = File.Create(path);
```
We should really use File.WriteAllText instead of these lines

JokeOfAllTrades replied:
Done

hexafluoride commented on SharpinitUnitTest/UnitRegistryTests.cs:
```diff
@@ -4,20 +4,33 @@
     {
 
+        static string path = ".\\test.service";
+        static string contents = "[Unit]\nDescription = Notepad\n\n[Service]\nExecStart = notepad.exe";
```
Maybe this could be structured like so?

    static string Contents = "[Unit]\n" + 
        "Description = Notepad\n" +
        "\n" + 
        "[Service]\n" + 
        "ExecStart = notepad.exe";

which VS supports formatting into natively, just hit Enter after the newlines

JokeOfAllTrades replied:
Done

hexafluoride reviewed (changes requested):
We should at least use File.WriteAllText

JokeOfAllTrades committed:
Fixed some minor issues

Made the test file string more readable and used File.WriteAllText instead of a file stream.
